src/client/stdlib/fileAttachment.js

```diff
@@ -28,9 +28,10 @@ async function dsv(file, delimiter, {array = false, typed = false} = {}) {
 }
 
 export class AbstractFile {
-  constructor(name, mimeType = "application/octet-stream") {
-    Object.defineProperty(this, "name", {value: `${name}`, enumerable: true});
+  constructor(name, mimeType, lastModified) {
```

Comment:
Looks like you (inadvertently?) dropped the default for `mimeType` here. I think we want to keep that (and I’m surprised there’s not a test). Also this is part of a public API so we wouldn’t want to remove the default in any case.

Reply:
Oops, good catch. I'll add a test!

Reply:
Ah now I remember—I wanted to change this so that it would be defined in the call to registerFile instead of in the client.

So, in render.ts we'd have:

```diff
-    mimeType: mime.getType(name) ?? undefined,
+    mimeType: mime.getType(name) ?? "application/octet-stream",
```

and then we don't need this default in the client (and it becomes possible to do a "build test").

Reply:
I've done that now; we can roll back and add the default to client/stdlib, but then I don't know how to test.

Reply:
Yes, please move the default back to the client.

Reply:
done